docs/getting-started/docker-playgrounds/conf/flink-conf.yaml

```diff
@@ -0,0 +1,7 @@
+jobmanager.rpc.address: jobmanager
```

Comment:
Since users have to copy this file anyway, maybe the initial version remains bare-bones and part of the walkthrough is filling it out. Ports and anything that needs to be coordinated with docker to work should be present, but savepoint/checkpoint options for instance.

Reply:
At a minimum, since this is an operations walkthrough it should walk through the configs.

Reply:
Good point.

Reply:
I thought about this again (see my post slack). For me this is not so much an operations walkthrough, but the description of a simple, working docker-compose setup to play around with. Therefore I don't have multiple steps for now.